test/common.js

```diff
@@ -445,3 +446,11 @@ exports.fileExists = function(pathname) {
     return false;
   }
 };
+
+exports.busyLoop = function busyLoop(time) {
```

Comment:
Good question.  :)  I'm glad to remove it until others need something similar.  My only worry is removing it means people will probably write their own stuff not knowing that we have a common need for a busy loop.  Your call.